index.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = exports = require('./dist/jmap-client.min.js');
```

Comment:
A personal question: What is the diff between exports and module.exports? and why we should do both of them?